GVRf/Framework/jni/objects/components/eye_pointee_holder.h

```diff
@@ -69,7 +69,7 @@ class EyePointeeHolder: public Component {
 private:
     bool enable_;
     glm::vec3 hit_;
-    std::vector<std::shared_ptr<EyePointee>> pointees_;
+    std::vector<EyePointee*> pointees_;
```

Comment:
> Just a quick question. I can see comments in other codes that the reason why you don't do delete with some std containers elements is because in Java, the finalize() will handle those pointers' content. So is pointees__?

I am _not_ sure I know what you are asking! My best guess is 'Are these `EyePointee`s leaked?'

I don't think so. The `GVREyePointeeHolder` has a list of `GVREyePointee`s: when it is garbage, so are the  `GVREyePointee`s.